monitoring/README.md

```diff
@@ -46,6 +46,85 @@ metrics from the workers and provide the `/metrics` endpoint for Prometheus.
    Pushgateway] to be set up and configured for scraping.
 3. Monitor some resources.
 
+### How to monitor the Openshift objects?
+
+1. [`kube-state-metrics`](https://github.com/kubernetes/kube-state-metrics)
+
+   - converts Kubernetes objects to metrics consumable by Prometheus
+   - not focused on the health of the individual Kubernetes components, but rather on the health of the various objects inside, such as deployments, nodes and pods
```

Comment:
does this imply it focuses on application pods, services, deployments, controllers etc. instead of the functional kubernetes parts such as nodes, container runtime, master api, container registry...

Reply:
yes, right